packages/swapper/src/api.ts

```diff
@@ -45,7 +45,13 @@ type ChainSpecificQuoteFeeData<T extends ChainId> = ChainSpecific<
 >
 
 export type QuoteFeeData<T extends ChainId> = {
+  minerFee: string // fee paid to the miner from the fee asset
```

Comment:
thoughts on using `networkFee` over `minerFee`? i feel like we don't really refer to this as a miner fee anywhere in our stack, but do have this concept already as a network fee so may align more with existing vernacular.

Reply:
Completely agree, `networkFee` is more agnostic.

Changed to `networkFee` in https://github.com/shapeshift/lib/pull/1038/commits/186019fa4f39fe917f8d6231e4322fdb49daf032.